manage.py

```diff
@@ -1,57 +1,149 @@
 #!/usr/bin/env python
 
-import json
+
+import tabulate
 import typer
-from typing import List
+from typing import List, Optional
 from typing_extensions import Annotated
 from sqlmodel import Session
 from sqlmodel.sql.expression import select
 from fastapi import Depends
 from fastapi.params import Security
 from app.models.users import User, UserScopes
+from app.auth.security import get_password_hash
 from app.db import get_session
 from app.main import create_app
 
+
 app = typer.Typer()
 
-session: Session = next(get_session())
+
+def init_session(typer_app: typer.Typer, session: Session) -> None:
+    typer_app.db_session = session
 
 
 @app.command()
-def user_scopes_add(
+def add_scopes(
     username: str, scope: Annotated[List[UserScopes], typer.Option()]
 ) -> None:
     statement = select(User).where(User.username == username)
-    user: User = session.exec(statement).first()
+    user: User = app.db_session.exec(statement).first()
     print(
         f"Replacing user {user.username} current scopes {user.scopes} with new scopes {scope}..."
     )
     user.scopes = scope
```

Comment:
Do we need confirmation message here and for the standard update-user as either could invalidate access to the API.

Reply:
Added confirmation message to update operation